meteor/lib/api/userActions.ts

```diff
@@ -394,6 +399,8 @@ export enum UserActionAPIMethods {
 	'switchRouteSet' = 'userAction.switchRouteSet',
 
 	'disablePeripheralSubDevice' = 'userAction.system.disablePeripheralSubDevice',
+
+	'getDebugStates' = 'userAction.playout.getDebugStates',
```

Comment:
I don't think this should be `userAction.playout.getDebugStates`. This is not a "playout" action, as far as the user is concerned. Maybe this should be a `userAction.debug.*`?